packages/services/f-test-data-creator/src/clients/basketApi.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios');
+const { getLanguageForTenant } = require('../configuration/tenants');
+
+module.exports = class BasketApiService {
+    constructor (configuration) {
+        this.basketUrl = configuration.Services.Basket.BaseAddress;
+        this.tenant = configuration.Tenant;
+        this.postcode = configuration.Services.Search.Postcode;
+    }
+
+    async createBasketForUser (basketInfo, timeout = 5000) {
+        const config = {
+            headers: {
+                'Content-Type': 'application/json',
+                'Accept-Language': getLanguageForTenant(this.tenant),
+                'Accept-Tenant': this.tenant,
+                Authorization: `Bearer ${basketInfo.authToken}`
```

Comment:
Can you make this key a string so it's consistent with the others?

Reply:
The other keys are string as they aren't valid JS variable names whilst this one is — I think the linter will complain if you try to use a string here also.

Reply:
> This is a very interesting approach on solving a problem we currently face. I will be following its development. Question is would that provide always accurate data structures, that stay in sync with real world example.

Thanks @zlatinivanov-je! Yeah that's true, we'll probably need to keep an eye on it and update when needed. It calls the real APIs to grab the data which in this sense keeps it accurate. I definitely feel we can do more custom logic in future to involve time of day, choose more specific restaurants and menus, to make it more in sync with a real user - this will probably be the next stage :)